lte/gateway/python/integ_tests/s1aptests/test_gateway_metrics_attach_detach.py

```diff
@@ -138,6 +137,12 @@ def test_gateway_metrics_attach_detach(self):
             )
             assert (val == v_spgw_delete_session + 1)
 
+        mme_new_association = self._getMetricValueGivenLabel(
+            "mme_new_association",
+            label_values_success,
+        )
+        assert (mme_new_association == mme_new_association_before + 1)
```

Comment:
What does this counter measure? I would have assumed that this value is 2 after the test.

Reply:
I am not completely sure ... I started my local test with +2, but it seems to be +1. The doc says "New SCTP association". I.e., maybe something like a session based counter.

Reply:
If the are sctp association 1 makes sense

Reply:
Yes, on new sctp association, for label "mme_new_association" count gets incremented.